mux/src/tmux_commands.rs

```diff
@@ -294,8 +1036,12 @@ impl TmuxCommand for Resize {
 #[derive(Debug)]
 pub(crate) struct CapturePane(TmuxPaneId);
 impl TmuxCommand for CapturePane {
-    fn get_command(&self) -> String {
-        format!("capturep -p -t %{} -e -C\n", self.0)
+    fn get_command(&self, _domain_id: DomainId) -> String {
+        const HISTORY_LINES: isize = -2000;
+        format!(
+            "capture-pane -p -t %{} -e -C -S {}\n",
+            self.0, HISTORY_LINES
```

Comment:
FWIW, you can slightly simplify this:

```suggestion
            "capture-pane -p -t %{} -e -C -S {HISTORY_LINES}\n",
            self.0
```